tokio-threadpool/src/worker/mod.rs

```diff
@@ -892,4 +892,12 @@ impl WorkerId {
     pub(crate) fn new(idx: usize) -> WorkerId {
         WorkerId(idx)
     }
+
+    /// Returns this identifier represented as an integer.
+    ///
+    /// Worker identifiers in a single thread pool are guaranteed to correspond to integers in the
+    /// range `0..pool_size`.
+    pub fn into_usize(&self) -> usize {
```

Comment:
This is still taking `&self`. That said, I would opt for `to_usize` for now (but it isn't critical either way).